annotated_types/test_cases.py

```diff
@@ -93,7 +92,7 @@ def cases() -> Iterable[Case]:
     yield Case(Annotated[str, at.Len(3, 5)], ('123', '1234'), ('', '1', '12', '12345', 'x' * 10))
     yield Case(Annotated[str, 3:5], ('123', '1234'), ('', '1', '12', '12345', 'x' * 10))
 
-    yield Case(Annotated[Dict[int, int], at.Len(2, 4)], ({1: 1, 2: 2}), ({}, {1: 1}, {1: 1, 2: 2, 3: 3, 4: 4}))
+    yield Case(Annotated[Dict[int, int], at.Len(2, 4)], [{1: 1, 2: 2}], [{}, {1: 1}, {1: 1, 2: 2, 3: 3, 4: 4}])
```

Comment:
I think these can stay as tuples

Reply:
Discussed in person, we'll make them all lists, maybe tomorrow